CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ Changelog
     and in `ISA`s (@ecpeterson, gh-1096, gh-1107, gh-1111).
 -   Removed the `tox.ini` and `readthedocs.yml` files (@karalekas, gh-1108).
 -   Type hints have been added to the `PauliSum` class (@rht, gh-1104).
+-   Some of the type hints have been added to the `Program` class (@rht, gh-1115)
```

Comment:
This will need to move up to probably the `v2.16` section once after I release pyQuil today.

Reply:
Waiting for the `v2.16` development version to appear in the changelog.

Reply:
Moved to `v2.16`.